src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -100,19 +101,24 @@ Status RedisStoreClient::AsyncGet(const std::string &table_name, const std::stri
 
 Status RedisStoreClient::AsyncGetAll(
     const std::string &table_name,
-    const SegmentedCallback<std::pair<std::string, std::string>> &callback) {
-  RAY_CHECK(0) << "Not implemented! Will implement this function in next PR.";
-  return Status::OK();
+    const MultiItemCallback<std::pair<std::string, std::string>> &callback) {
+  RAY_CHECK(callback);
+  std::string match_pattern = table_name + "*";
```

Comment:
Should add a separator after table_name.  Because otherwise this pattern will also match other tables that have the same prefix (`actor` and `actor_checkpoint` for example).

Reply:
fixed